net-vpn/tor/tor-0.3.2.9.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="A virtual layer that helps you improve privacy and anonymity"
+DESCRIPTION="Tor is a network of virtual tunnels that allows people and groups to improve \
+their privacy and security on the Internet. It also enables software \
+developers to create new communication tools with built-in privacy features. \
+Tor provides the foundation for a range of applications that allow \
+organizations and individuals to share information over public networks \
+without compromising their privacy."
+HOMEPAGE="https://www.torproject.org"
+COPYRIGHT="
+	2001-2004, Roger Dingledine
+	2004-2006, Roger Dingledine, Nick Mathewson
+	2007-2018, The Tor Project, Inc.
+	"
+LICENSE="Tor"
+REVISION="1"
+SOURCE_URI="https://www.torproject.org/dist/tor-$portVersion.tar.gz"
+CHECKSUM_SHA256="435a7b91aa98d8b1a0ac1f60ca30c0ff3665b18a02e570bab5fe27935829160f"
+PATCHES="tor-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	tor = $portVersion
```

Comment:
```diff
 PROVIDES="
-	tor = $portVersion
+	tor = $portVersion$secondaryArchSuffix
```

Reply:
wat?

Reply:
Oops, I just fixed my comment.

Reply:
Shouldn't that be "tor$commandSuffix = $portVersion" then?

Reply:
Nope. `$commandSuffix` is only used for `cmd:` entries.

The provides for the package name should always end with `$secondaryArchSuffix`, even if we use the `commandSuffix`.

Reply:
Oops, my bad :)